api/openapi-spec/swagger.json

```diff
@@ -3291,6 +3291,11 @@
      "volumeName"
     ],
     "properties": {
+     "bootOrder": {
+      "description": "BootOrder is an integer value used to determine ordering of boot devices\n+optional",
```

Comment:
Can you also add something like `\nMust be a value greater or equal 1.` Also if lower values are tried first or later.

Reply:
And also how disks without a bootOrder are treated if some have one.